### DISCUSSION TITLE
Block certain notify messages, when noice handles vim.notify

### DISCUSSION THREAD

**Discussion by ghillb** [folke/noice.nvim#125] [Q&A] [answered]
Hi, 

I have the following issue:

Currently, I'm using `vim.notify` to block a message from `clangd` like so:

```lua
local notify = vim.notify
vim.notify = function(msg, ...)
  if msg:match("warning: multiple different client offset_encodings") then
    return
  end
  notify(msg, ...)
end
```

When I use noice as notify handler, it throws the warning `vim.notify has been overwritten by another plugin?`.

How would I achieve the same blocking of the message via noice?

**Comment by folke** [comment, marked as answer]
See here https://github.com/folke/noice.nvim/issues/118
